cernopendata/templates/cernopendata_records_ui/records/record_detail.html

```diff
@@ -192,6 +192,10 @@ <h2>System details</h2>
                 <label>Recommended release for analysis:</label>
                 {% endif %}
                 <span>{{record.system_details.release}}</span>
+                {% for container_image in record.system_details.container_images %}
+                    <p>{{ container_image.registry.capitalize() }} container image FQIN: {{ container_image.name }}</p>
+                {% endfor %}
+                <p>To learn more about running CMS analysis code using Docker you can refer to this <a href="/docs/cms-guide-docker">guide</a> </p>
```

Comment:
BTW this text ("To learn more...") will appear also for datasets that don't have any container images defined.

You may want to wrap it under `{% if record.system_details_container_images %}` conditional clause so that it appears only when the record contains those fields